packages/vike-react-styled-jsx/Wrapper.tsx

```diff
@@ -0,0 +1,21 @@
+export { Wrapper }
+
+import React, { type ReactNode } from 'react'
+import { StyleRegistry, createStyleRegistry } from 'styled-jsx'
+import { usePageContext } from 'vike-react/usePageContext'
+
+function Wrapper({ children }: { children: ReactNode }) {
+  const pageContext = usePageContext()
+
+  if (isBrowser() || pageContext.config.styledJsx === null) return <>{children}</>
```

Comment:
Instead of using `isBrowser()` how about we use `meta` to change Wrapper to be server-side only?

Loading `createStyleRegistry()` on the server-side is maybe a no-go. Intervention++.

Reply:
> Loading `createStyleRegistry()` on the server-side is maybe a no-go.

Sorry, I don't get it 😅

Reply:
Ah, sorry, I meant client-side.

Bottom line: `createStyleRegistry()` is meant for the server-side, correct? We don't need to import it on the client-side right? Will it bloat the client-side?

Reply:
Why does the `+Wrapper` still loads on the client side even though we've change meta env ?
Please take a look on the cli [error](https://github.com/vikejs/vike-react/actions/runs/12338722643/job/34434049230?pr=158):

```
"Error: [vike][Wrong Usage] pageContext.styledJsx isn't defined on the client-side,
```

Reply:
It's possibly a Vike bug, I'll have a look at it later. In the meantime let's use `'styledJsx' in pageContext` to avoid the warning (see the link in the warning) so that we can merge before I fix the bug.

Let me know if the PR is ready for review.

Reply:
Okay, thank you. 
The e2e tests are green now. As for the test:types error, I’m confident we can ignore it. 
I’ll go ahead and revert the `examples/full`, and then it’ll be ready for review 👍